homeassistant/components/homekit/__init__.py

```diff
@@ -0,0 +1,143 @@
+"""Support for Apple Homekit.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/homekit/
+"""
+import asyncio
+import logging
+
+import voluptuous as vol
+
+from pyhap.accessory_driver import AccessoryDriver
```

Comment:
Imports of dependencies should happen inside functions. They are not allowed to be in the main level because of how dependency installation works.

To work around this, you can move it to new files (like you already did), but then also only import those files inside functions.

```python
def async_setup(hass, config):
    from .accessories import HomeBridge
```